devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/README.md

```diff
@@ -1,10 +1,11 @@
-# OSRAM Lightify Tunable 60 White
+# ZigBee White Color Temperature Bulb
 
 
 
 Works with: 
 
 * [OSRAM Lightify Tunable 60 White](http://www.osram.com/osram_com/tools-and-services/tools/lightify---smart-connected-light/lightify-for-home---what-is-light-to-you/lightify-products/lightify-classic-a60-tunable-white/index.jsp)
+* [Osram RT5/6 Tunable White](https://www.smartthings.com/works-with-smartthings/light-bulbs/osram-lightify-rt56-tunable-white)
```

Comment:
In `zigbee-white-color-temperature-bulb.groovy`, let's change the devoiceJoinName so it matches to the WWST names